client/app/scripts/superdesk-archive/views/list.html

```diff
@@ -2,7 +2,7 @@
 	<div class="button-stack right-stack">   
 	    <div class="navbtn dropdown strict highlights-dropdown" 
             dropdown sd-package-highlights-dropdown 
-            ng-if="type === 'archive' && selected.desk">
```

Comment:
I think you can fix it if you add `$scope.selected = active;` to [controller](https://github.com/superdesk/superdesk/blob/master/client/app/scripts/superdesk-archive/controllers/list.js#L107-L114)